docs/collecting-data/collecting-from-own-applications/python-tracker/emitters/index.md

```diff
@@ -46,7 +51,7 @@ def __init__(
 
 - `protocol`
 
-`protocol` defaults to "http" but also supports "https".
+`protocol` defaults to "https" but also supports "http". **_\*\*(Since v0.12.0_)**
```

Comment:
```suggestion
`protocol` defaults to "https" but also supports "http".

:::note Prior to v0.12.0

In older versions, `protocol` defaulted to `http`.

:::
```

Reply:
Feels like a bit too much having this note right after the previous note which said the same thing.